Makefile

```
@@ -199,3 +167,25 @@ adr-gen:
 	@echo "--> Downloading ADR template"
 	@curl -sSL https://raw.githubusercontent.com/celestiaorg/.github/main/adr-template.md > docs/architecture/adr-template.md
 .PHONY: adr-gen
+
+## goreleaser: list goreleaser options and checks if goreleaser is installed
+goreleaser: Makefile
```

[question] Is the `Makefile` command a prerequisite of `goreleaser`? I don't think a `Makefile` command exists.

```suggestion
goreleaser:
```

---

This passes the Makefile in for the `sed` command to parse. This is how the `help` command works.

---

Thanks! Resolving.